easybuild/easyconfigs/n/numactl/numactl-2.0.14-GCCcore-10.3.0.eb

```diff
@@ -0,0 +1,37 @@
+##
+# Author:    Robert Mijakovic <robert.mijakovic@lxp.lu>
+##
```

Comment:
So.. maybe it's just me being picky here, but I really wish no extra comments were added to easyconfigs like this. This adds extra 3 lines of diff's when using `eb --review-pr` and if this is present in any version, then it's going to be there in all future PRs for this software, adding a bunch of extra diffs.
I know lots of various easyconfigs unfortunately already has comments like these, but, since this PR introduces 20 additional ones to otherwise clean easyconfigs I just felt i had to mention it.

Reply:
@Micket Some people have good reasons for adding headers like this, and I don't think it's a big issue...